src/components/molecules/LoginStatus/LoginStatus.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import { Header } from 'nhsuk-react-components';
+import React, { useEffect, useState } from 'react';
+import { getBasePath } from '@utils/get-base-path';
+import { usePathname } from 'next/navigation';
+import { fetchAuthSession } from '@aws-amplify/auth';
+import { useAuthenticator } from '@aws-amplify/ui-react';
+
+export const LoginStatus = () => {
+  const pathname = usePathname() ?? '/';
+  const { authStatus } = useAuthenticator();
+  const [loggedInUser, setLoggedInUser] = useState<string>();
+
+  useEffect(() => {
+    fetchAuthSession().then((session) => {
+      setLoggedInUser(session.tokens?.idToken?.payload?.email?.toString());
+    });
+  }, [authStatus]);
+
+  if (authStatus === 'authenticated') {
```

Comment:
Is there an enum to reference?

Reply:
no, Amplify defines it as a type, not an enum
![image](https://github.com/user-attachments/assets/d7077434-9713-49a4-945f-d22ba98ea500)